fvm/evm/offchain/sync/replay.go

```diff
@@ -107,6 +109,20 @@ func replayTransactionExecution(
 		ctx.ExtraPrecompiledContracts = precompiles.AggregatedPrecompiledCallsToPrecompiledContracts(pcs)
 	}
 
+	if ctx.BlockNumber == 8008603 && ctx.ChainID == types.FlowEVMTestNetChainID {
```

Comment:
It's the height where testnet was sporked, with the introduction of https://github.com/onflow/flow-go/blob/master/fvm/evm/types/address.go#L40-L41, as the `coinbase` address. Basically the `miner` of each EVM block, who gets the rewards.

Reply:
can we add this as a comment for this if statement, so its easier to find next time?

Reply:
Sorry, this PR is actually redundant now. The root cause of this was different.

Reply:
This is fixed in EVM Gateway, with this change: https://github.com/onflow/flow-evm-gateway/pull/659/files#diff-b3cc424323a936972f841745ae20117fcd028cecb7305a8ee4054023ca021e63R22-R25

Reply:
yeah, we better add comments to explain what is this specific height, and why do we need to delete an account.

Otherwise, we will always get confused whenever reading this code.